components/authz-service/storage/postgres/migration/sql/58_use_project_db_id_in_references_of_projects.up.sql

```diff
@@ -0,0 +1,159 @@
+BEGIN;
+
+ALTER TABLE iam_statement_projects
+    DROP CONSTRAINT iam_statement_projects_project_id_fkey;
+
+ALTER TABLE iam_statement_projects RENAME COLUMN project_id TO project_temp_id;
+
+ALTER TABLE iam_statement_projects ADD COLUMN project_id SERIAL;
```

Comment:
Prob don't want `SERIAL` bc we're manually populating it

Reply:
Isn't this even kind of wrong? 😉 Should be INTEGER, I suppose. There's nothing _serial_ about it; it's not auto-incrementing ... (ref: https://www.postgresql.org/docs/9.1/datatype-numeric.html)

Reply:
Looks like we did that wrong elsewhere as well. I'll fix here for starters.